diff --git a/node_modules/expo-modules-core/ios/Worklets/EXJavaScriptSerializable.mm b/node_modules/expo-modules-core/ios/Worklets/EXJavaScriptSerializable.mm
index 93bf657..3d9e117 100644
--- a/node_modules/expo-modules-core/ios/Worklets/EXJavaScriptSerializable.mm
+++ b/node_modules/expo-modules-core/ios/Worklets/EXJavaScriptSerializable.mm
@@ -48,6 +48,8 @@ EXSerializableValueType toObjCValueType(worklets::Serializable::ValueType type)
       return EXSerializableValueTypeSynchronizable;
     case worklets::Serializable::ValueType::CustomType:
       return EXSerializableValueTypeCustom;
+    default:
+      return EXSerializableValueTypeUndefined;
   }
 }
 
